README.md

```diff
@@ -223,9 +241,11 @@ App behavior parameters
   - Use the environment variable `ROOSEVELT_ENABLE_VALIDATOR` to force the validator on or off regardless of dev or prod mode
   - e.g. force validator off in dev mode regardless of app settings:  `export ROOSEVELT_ENABLE_VALIDATOR=false && npm run dev`
   - e.g. force validator on in prod mode regardless of app settings:  `export ROOSEVELT_ENABLE_VALIDATOR=true && npm run prod`
-- `htmlValidator`: Params to send to [html-validator](https://github.com/zrrrzzt/html-validator#usage) (if `enableValidator` is set to true). When `suppressWarnings` is set to true validation warnings will be hidden and only errors will be shown.
-  - Default:  `{port: '8888', format: 'text', suppressWarnings: false}`
+- `htmlValidator`: Params to send to [html-validator](https://github.com/zrrrzzt/html-validator#usage) (if `enableValidator` is set to true). When `suppressWarnings` is set to true validation warnings will be hidden and only errors will be shown. When `separateProcess` is turned to true the HTML validator will run separately from the node process.
+  - Default:  `{port: '8888', separateProcess: false, format: 'text', suppressWarnings: false}`
   - Can be disabled for individual requests by sending the request header `Partial` with the value set to `true` or by passing `_disableValidator` to the model and setting it to `true`.
+  - e.g. force validator to detach regardless of app settings: `export ENABLE_DETACH=true && npm run dev`
```

Comment:
follow the convention above with another bullet explaining the env var in general:

- Use the environment variable `ROOSEVELT_ENABLE_DETACH` to force the validator to be an attached or detached process